schemas/com.snowplowanalytics.mobile/application/jsonschema/1-0-0

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an application context",
```

Comment:
Also, not very descriptive. Is it something to uniquely identify an application? What kind of application? Mobile? Desktop? Server?

Reply:
will amend